packages/backend/src/graphql/project/mutations/projectUpdateMutationField.ts

```diff
@@ -37,6 +37,33 @@ builder.mutationField('projectUpdate', (t) =>
           organizationId,
         },
       })
+
+      if (organizationId) {
+        const projectMemberships = await prisma.projectMembership.findMany({
+          where: { projectId: id.toString() },
+        })
+
+        await Promise.all(
+          projectMemberships.map(async (membership) => {
+            await prisma.organizationMembership.upsert({
```

Comment:
can we use createMany instead? Create all users in organizationMemberships that are members in projectMemberships that do not exist in organization